settings.gradle

```diff
@@ -1,3 +1,4 @@
+include ':xm-commons-communication'
```

Comment:
we don't need separate project, put to messaging commons

Reply:
Done